### DISCUSSION TITLE
Steam controller bug

### DISCUSSION THREAD

**Discussion by UnrealisticDino** [canonical/steam-snap#268] [General]
Steam does not recognize that the controller is connected which means I can't remap inputs. With the adapter the controller does work mostly as expected. In Mighty No. 9 and 10 Second Ninja touching up or right on the left touch pad acts like holding right on the left analog stick until the touch pad is touched again on either the left or down areas. In A Story About My Uncle the right trigger doesn't work. If the controller is in Bluetooth mode the controller does not change inputs when in game and instead keeps behaving like a keyboard and mouse.

**Comment by ashuntu** [comment]
I moved this to issue https://github.com/canonical/steam-snap/issues/280.
